libs/blocks/preflight/panels/general.js

```diff
@@ -72,8 +72,12 @@ function findLinks(selector) {
   return [...document.body.querySelectorAll(selector)]
     .reduce((links, el) => {
       const url = getUrl(el);
-      if (!hrefs.includes(url.href)) {
-        hrefs.push(url.href);
+      const baseUrl = url.origin + url.pathname;
+      if (url.pathname.includes('/tools/caas')) {
+        return links;
+      }
```

Comment:
A Milo-specific nit: for brevity, when there's a single declaration inside a conditional, we drop the brackets and have everything on one line.